Dockerfile

```diff
@@ -24,7 +24,7 @@ ENV CARGO_INSTALL_ROOT=${HOME}/.cargo
 ENV PATH ${PATH}:${HOME}/.cargo/bin
 
 # Install a tagged version of jsonschema-transpiler
-RUN cargo install jsonschema-transpiler --version 1.0.0
+RUN cargo install jsonschema-transpiler --version 1.2.0
```

Comment:
I'm assuming the jsonschema-transpiler changes will be deployed as 1.2.0

Reply:
Yes, the package has been [tagged](https://github.com/mozilla/jsonschema-transpiler/releases/tag/v1.2.0) and [deployed to cargo](https://crates.io/crates/jsonschema-transpiler). It's ready for testing and use.